src/main/kotlin/com/exactpro/th2/common/value/ValueUtils.kt

```diff
@@ -86,3 +125,17 @@ fun IntArray.toValue(): Value = toTypedArray().toValue()
 fun LongArray.toValue(): Value = toTypedArray().toValue()
 fun FloatArray.toValue(): Value = toTypedArray().toValue()
 fun DoubleArray.toValue(): Value = toTypedArray().toValue()
+
+
+fun Iterator<*>.toListValue() : ListValue = listValue().also { builder -> while(hasNext()) { next()?.also { builder.addValues(it.toValue()) } } }.build()
```

Comment:
```suggestion
fun Iterator<*>.toListValue(): ListValue = listValue().also { list -> 
    forEach { 
        it?.toValue().run(list::addValues)
    }
}.build()
```
Also - why do we ignore null-values?

Reply:
This behavior was in prev version. And null values do not have any useful information. If user want to add null value, he can use `add` method.